server/modules/blendsubs/blendsubs.js

```diff
@@ -0,0 +1,48 @@
+var fs = require('fs');
+var threshold = '0.2';
+
+var file = 'samples/the-walking-dead_pt-br.srt',
+    destination = 'samples/obfuscated.srt';
+
+console.log('Reading str file: ' + file);
+fs.readFile(file, 'utf8', function(err, text) {
+  if (err) throw err;
+  console.log('Srt file read');
+
+  var lines = text.split('\n'),
+      changedWords = [];
+
+  for (var i = 0; i < lines.length; i++) {
+    if (lines[i].length < 10 || lines[i].indexOf('-->') > -1) {
+      continue;
+    }
+
+    var words = lines[i].split(' ');
+
+    for (var j = 0; j < words.length; j++) {
+      if (words[j].length < 3 || /[A-Z<>]/.test(words[j])) {
```

Comment:
word filtering - if the word has at least one uppercase char or HTML tags, it won't be considered